_gsdocs-orgs/2019/uoslo.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "University of Oslo"
+author: "Cédric Serfon"
+layout: default
+organization: UiO
```

Comment:
The acronym `UiO` should be consistent with the one specified in the `organization` field inside the proposal description (`UOslo`), otherwise the organization will not appear as part of the proposal. I would use here: `UOslo` which is also consistent with the acronym used for this university in the GSoC program.

```suggestion
organization: UOslo
```